src/Avalonia.Controls/ThemeVariantScope.cs

```diff
@@ -7,6 +7,23 @@ namespace Avalonia.Controls
     /// </summary>
     public class ThemeVariantScope : Decorator
     {
+        /// <summary>
+        /// Defines the <see cref="StyledElement.ActualThemeVariant"/> property.
+        /// </summary>
+        public static readonly StyledProperty<ThemeVariant> ActualThemeVariantProperty =
+            AvaloniaProperty.Register<ThemeVariantScope, ThemeVariant>(
+                nameof(ThemeVariant),
+                inherits: true,
+                defaultValue: ThemeVariant.Light);
+
+        /// <summary>
+        /// Defines the <see cref="RequestedThemeVariant"/> property.
+        /// </summary>
+        public static readonly StyledProperty<ThemeVariant?> RequestedThemeVariantProperty =
+            AvaloniaProperty.Register<ThemeVariantScope, ThemeVariant?>(
+                nameof(ThemeVariant),
+                defaultValue: ThemeVariant.Default);
+        
```

Comment:
I think property names are wrong? Should be `"ActualThemeVariant"` and `nameof(RequestedThemeVariant)`.

Also, a get-only CLR property for `ActualThemeVariant` could be added here I guess? And in that case it would be `nameof(ActualThemeVariant)` in the property name.

The XML doc also has to be fixed, as `StyledElement.ActualThemeVariant` doesn't exist now.

Reply:
Yes, it's wrong here as well.
I need to fix that.